br/pkg/restore/client.go

```diff
@@ -1588,7 +1588,10 @@ func (rc *Client) WaitForFilesRestored(ctx context.Context, files []*backuppb.Fi
 		fileReplica := file
 		rc.workerPool.ApplyOnErrorGroup(eg,
 			func() error {
-				defer updateCh.Inc()
+				defer func() {
+					log.Info("import sst files done", logutil.Files(files))
```

Comment:
Will this be too verbose?

Reply:
tidb backup also have the same log. I think it's okay when restore a large amount data. it's better to know br is importing rather than nothing output.